js/modules/tamagotchi.js

```diff
@@ -9,10 +9,51 @@ export default class Tamagotchi {
 
   displayHealth = (elementSelector) => {
     const displayElement = document.querySelector(elementSelector);
-    displayElement.innerText = this.energy.value;
+    displayElement.innerText = this.health.value;
   };
 
+  decreaseParameters = () => {
+
+    const healthInterval = setInterval(() => {
+      if (this.hunger.value <= 0 || this.energy.value <= 0) {
+        if (this.health.value > 0) {
+          this.health.value -= 1
+        } else {
+          clearInterval(healthInterval)
+        }
+      }
+    }, 1000)
+
+    const energyInterval = setInterval( () => {
+      if (this.energy.value > 0 && this.fun.value === 0) {
+        this.energy.value -=2
+      } else if (this.energy.value > 0) {
+        this.energy.value -=1
+      } else {
+        clearInterval(energyInterval)
+      }
+    }, 2000)
+
+    const hungerInterval = setInterval( () => {
+      if (this.hunger.value > 0) {
+        this.hunger.value -= 1
```

Comment:
**issue:** You are decreasing values, but you don't display changed value in your UI - check `this.displayHealth(healthElement)` for the reference. You should have similar methods for other parameters, and the call such method in your interval